### DISCUSSION TITLE
feat: `--dry-run` option for `apply`

### DISCUSSION THREAD

**Discussion by jjangga0214** [helmfile/helmfile#1024] [Ideas]
Hi!

I think `--dry-run` option for `apply` would be cool.

```bash
# calling helm's --dry-run option interally 
helmfile apply --dry-run 
```
It'd make helmfile more consistent with helm.
Or.. is this option intentionally not provided?

Thanks :)

**Comment by yxxhero** [comment]
@jjangga0214 PR is welcome.

**Reply by jjangga0214** [reply]
Thanks for the prompt reply!

But I've never written a single line of go.
I hope someone who also thinks this feature is valuable to contribute.

**Reply by yxxhero** [reply]
see: https://github.com/helmfile/helmfile/pull/1050
